Dockerfile

```diff
@@ -1,4 +1,11 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
 
-RUN yum -y install java-1.8.0-openjdk
+
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_base_java8
```

Comment:
@naveen-armedia  please take a look at @fabricetriboix's Dockerfiles and see if there's any other labels we should add.  I noticed in the meeting earlier you also had maintainer as well.